pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-standalone/src/main/java/org/apache/pinot/plugin/ingestion/batch/standalone/SegmentGenerationJobRunner.java

```diff
@@ -70,6 +73,7 @@ public class SegmentGenerationJobRunner implements IngestionJobRunner {
   private CountDownLatch _segmentCreationTaskCountDownLatch;
   private Schema _schema;
   private TableConfig _tableConfig;
+  private Queue<Pair<Exception, String>> _failures;
```

Comment:
The `Exception` is not used. Consider changing it to
```suggestion
  private Queue<String> _failureFilePaths;
```